README.md

```diff
@@ -214,6 +216,23 @@ day of week    0-7 (0 or 7 is Sunday, or use names)
 
 - `addCallback`: Permits addition of `onTick` callbacks.
 
+#### Properties
+
+- `isRunning`: [READ-ONLY] Indicates if a callback is currently executing. This is different from the `running` property which indicates if the cron schedule is active.
```

Comment:
I'm thinking we should differentiate these more with more descriptive naming. we should probably maintain `job.running` for backwards compatibility (even though that property is not documented and technically not officially supported). something as simple as `job.isCallbackRunning` instead of `job.isRunning` should work.

Reply:
I agree. `job.running` and `job.isRunning` are definitely easy to confuse. Even if we change it to `job.isCallbackRunning`, we’ll still need documentation on how to use it.